nodebuilder/config.go

```diff
@@ -72,8 +72,9 @@ func LoadConfig(path string) (*Config, error) {
 
 // Encode encodes a given Config into w.
 // TODO(@Wondertan): We should have a description for each field written into w,
-//  so users can instantly understand purpose of each field. Ideally, we should have a utility program to parse comments
-//  from actual sources(*.go files) and generate docs from comments. Hint: use 'ast' package.
+//
+//	so users can instantly understand purpose of each field. Ideally, we should have a utility program to parse comments
+//	from actual sources(*.go files) and generate docs from comments. Hint: use 'ast' package.
```

Comment:
format issue?

Reply:
nvm its my ide that's running the new gofmt

Reply:
fixed but had to be removed from the godoc like in #1067